README.md

```diff
@@ -5,10 +5,11 @@ The equality saturation engine used is [Egglog](https://github.com/egraphs-good/
 ## Getting Started
 
 ### LLVM
-Clone [LLVM](https://github.com/llvm/llvm-project) with tag `llvmorg-18.1.4` or commit `e6c3289804a67ea0bb6a86fadbe454dd93b8d855`. DialEgg has been tested with this version of LLVM only.
+Clone [LLVM](https://github.com/llvm/llvm-project) with commit `7b074fc9362a4a6a5f172dd8936a22fb25eff96b`.
 
 ```bash
-git clone -b llvmorg-18.1.4 --depth 1 https://github.com/llvm/llvm-project.git llvm
```

Comment:
I used `--depth 1` to keep the size of the repo reasonable by only cloning the latest commit on the main branch. Is there a reason for cloning everything?

Also `-b llvmorg-18.1.4` is a readable way to know which llvm version we are cloning, instead of a commit hash. We can use `-b llvmorg-20.1.6` as ther latest version.

Reply:
I agree that not cloning the full history would be ideal for both disk space and cloning time, and that using a tag is more readable than a hash. Unfortunately, since MLIR is constantly changing all the time, keeping a project pinned to a specific version is critical, and the version of MLIR that the other projects I'm trying to sync with are using is so recent that it doesn't have a tag yet.